src/main/java/com/aws/greengrass/clientdevices/auth/ClientDevicesAuthService.java

```diff
@@ -275,28 +282,35 @@ private void updateDeviceGroups(WhatHappened whatHappened, Topics deviceGroupsTo
         }
     }
 
-    private void updateCAType() {
-        // NOTE: This entire method will be moved out of here and will become part of a workflow.
-        String thingName = Coerce.toString(deviceConfiguration.getThingName());
+    @SuppressWarnings("PMD.AvoidCatchingGenericException")
```

Comment:
The reason we are suppressing here is because we are using the retries that throw a generic exception if it fails.